docs/API.md

```diff
@@ -128,30 +128,30 @@ Lists all buckets.
 
 | Param  | Type  | Description  |
 |---|---|---|
-|`bucketList`  | _[]BucketInfo_  | Lists bucket in following format shown below: | 
+|`bucketList`  | _[]BucketInfo_  | Lists buckets |
```

Comment:
"Lists buckets" means this is doing something. In-fact this is return value so it should be "List of all buckets"